doc/api/next_api_changes/behavior/22135-JSS3.rst

```diff
@@ -0,0 +1,8 @@
+Figure & subfigure clearing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Fixed a bug in which ``Figure.clf()`` and ``Figure.clear()`` were broken 
+when ``Figure`` contained axes not stored in ``Figure._localaxes``.  Moved the
```

Comment:
`Figure._localaxes` is not public API, and so doesn't belong in the API change notes. I'm not sure that this change requires a note at all, tbh.